### PR TITLE
Brazilian portuguese translation for SQUAD UI

### PR DESCRIPTION
**Pull request by chaws**
Done translating to Brazilian portuguese. @terceiro could you double check it briefly? Some words are just too tricky to translate.

Also, I spotted a small typo ("projec" instead of "project') in [delete.jinja2](https://github.com/Linaro/squad/blob/master/squad/frontend/templates/squad/project_settings/delete.jinja2#L15). When I ran `./scripts/update-translation-files`, both polish and portuguese as well as *pot files got changed. Maybe we should add that step to the documentation as well.

### CODE REVIEW

**Comment by mwasilew** [comment]
There are also 1 or 2 cases where the strings only differ by punctuation "." or ":". I'm not sure we should remove punctuation from strings as it might bite us back with RTL languages.

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,1008 @@
...
+#: squad/frontend/templates/squad/build.jinja2:161
+#: squad/frontend/templates/squad/metrics.jinja2:16
+#: squad/frontend/templates/squad/test_run.jinja2:80
+#: squad/frontend/templates/squad/test_run.jinja2:109
+msgid "Environment"
+msgstr "Ambiente"
+
+#: squad/frontend/templates/squad/metrics.jinja2:19
+msgid "Select environment(s)"
+msgstr "Selecione os ambiente(s)"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: o(s)

File: squad/core/locale/pt_BR/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+msgid "Is archived"
+msgstr "Está arquivado"
+
+#: squad/core/models.py:1122
+msgid "All builds"
+msgstr "Todas as builds"
+
+#: squad/core/models.py:1123
+msgid "Only on change"
+msgstr "Apenas quando houver modificação"
```
Comment (terceiro) [inline squad/core/locale/pt_BR/LC_MESSAGES/django.po]: mudança é melhor

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,1008 @@
...
+msgstr "Execução de teste"
+
+#: squad/frontend/templates/squad/test_run.jinja2:104
+msgid "Metrics suite"
+msgstr "Suite de métricas"
+
+#: squad/frontend/templates/squad/build.jinja2:153
+#: squad/frontend/templates/squad/test_run.jinja2:119
+msgid "Test runs"
+msgstr "Execuçõe de teste"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: falta um s em execuções

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,1008 @@
...
+msgid "Metadata file"
+msgstr "Arquivo de meta-dados"
+
+#: squad/frontend/templates/squad/test_run.jinja2:75
+msgid "Test suite"
+msgstr "Suite de teste"
+
+#: squad/frontend/templates/squad/test_run.jinja2:90
+msgid "Test run"
+msgstr "Execução de teste"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: testes

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,1008 @@
...
+msgid "Metrics file"
+msgstr "Arquivo de métricas"
+
+#: squad/frontend/templates/squad/test_run.jinja2:52
+msgid "Metadata file"
+msgstr "Arquivo de meta-dados"
+
+#: squad/frontend/templates/squad/test_run.jinja2:75
+msgid "Test suite"
+msgstr "Suite de teste"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: testes

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po (hunk 3 above)
Comment (terceiro) [reply squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: testes tb

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -0,0 +1,1008 @@
...
+msgstr "Regressões"
+
+#: squad/frontend/templates/squad/_regressions_and_fixes.jinja2:7
+#, python-format
+msgid "Regressions found on <strong>%s</strong>:"
+msgstr "Regressões encontradas em <strong>%s<strong>:"
+
+#: squad/frontend/templates/squad/_regressions_and_fixes.jinja2:21
+msgid "Click to display fixes"
+msgstr "Clique para mostrar os fixes"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: consertos?

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -0,0 +1,1008 @@
...
+msgstr "Fixes"
+
+#: squad/frontend/templates/squad/_regressions_and_fixes.jinja2:25
+#, python-format
+msgid "Fixes found on <strong>%s</strong>:"
+msgstr "Fixes encontrados em <strong>%s<strong>:"
+
+#: squad/frontend/templates/squad/build.jinja2:63
+msgid "Test Run"
+msgstr "Execução de teste"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: testes

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -0,0 +1,1008 @@
...
+
+#: squad/frontend/templates/squad/build.jinja2:71
+msgid "Test Results"
+msgstr "Resultados de teste"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: testes

**Comment by terceiro** [comment]
I also found both "Test run" and "Test Run" (different capitalization). we should probably fix that

@chwas wrt typos in the templates: yes we need to document what to do. I will give it a shot

**Comment by chaws** [comment]
@terceiro Thanks for your comments. Applied all of them.

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -0,0 +1,1008 @@
...
+msgid "Regressions found on <strong>%s</strong>:"
+msgstr "Regressões encontradas em <strong>%s<strong>:"
+
+#: squad/frontend/templates/squad/_regressions_and_fixes.jinja2:21
+msgid "Click to display fixes"
+msgstr "Clique para mostrar os consertos"
+
+#: squad/frontend/templates/squad/_regressions_and_fixes.jinja2:23
+msgid "Fixes"
+msgstr "Fixes"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: Consertos

File: squad/frontend/locale/pt_BR/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -0,0 +1,1008 @@
...
+
+#: squad/frontend/templates/squad/_regressions_and_fixes.jinja2:25
+#, python-format
+msgid "Fixes found on <strong>%s</strong>:"
+msgstr "Fixes encontrados em <strong>%s<strong>:"
```
Comment (terceiro) [inline squad/frontend/locale/pt_BR/LC_MESSAGES/django.po]: Consertos

**Comment by chaws** [comment]
Also I've pluralized alll "teste" followed by "de", e.g., "job de testes", idk it sounds better to me

**Pull request merged**